stable/wordpress/values-production.yaml

```diff
@@ -84,45 +84,44 @@ externalDatabase:
 ## MariaDB chart configuration
 ##
 mariadb:
-  ## Whether to use the database specified as a requirement or not.
-  ## If you want to use an external database, set this to false and supply details to externalDatabase above
+  ## Whether to deploy a mariadb server to satisfy the applications database requirements. To use an external database set this to false and configure the externalDatabase parameters
   enabled: true
+  ## Disable MariaDB replication
```

Comment:
Shouldn't we enabled replication on Production values?

Reply:
@beltran-rubo @prydonius @vtuson  thoughts?

Reply:
@javsalgar correct me if I'm wrong but we'll finally leave this as it's since WP does not require MariaDB replicas. Isn't it?

Reply:
Exactly

Reply:
All right! Let's go ahead then